posthog/temporal/ducklake/ducklake_copy_data_imports_workflow.py

```diff
@@ -248,7 +241,7 @@ def _detect_data_imports_partition_column(table_uri: str) -> str | None:
 
 def _fetch_delta_partition_columns(table_uri: str) -> list[str]:
     """Fetch partition columns from Delta table metadata."""
-    options = _get_delta_storage_options()
+    options = get_deltalake_storage_options()
```

Comment:
I may be missing something but it seems like after we call this, DeltaTable call below won't get any creds? `to_deltalake_options` is going to return an instance of `DuckLakeStorageConfig` with no key or secret. How does deltaTable use credential chain?